### PR TITLE
Initial release process

### PR DESCRIPTION
**Pull request by rubenvp8510**

### CODE REVIEW

**Review by frzifus — COMMENTED** [review]
Should we also add something like this [changelog.yaml](https://github.com/open-telemetry/opentelemetry-operator/blob/c4f8ffe8e221d9e0f7ce134de96f7aee02c3678d/.github/workflows/changelog.yaml) workflow?

**Comment by rubenvp8510** [comment]
> Should we also add something like this [changelog.yaml](https://github.com/open-telemetry/opentelemetry-operator/blob/c4f8ffe8e221d9e0f7ce134de96f7aee02c3678d/.github/workflows/changelog.yaml) workflow?

I would like to, but first want to have this in place and then I will enforce the changelog stuff in a separate PR, WDYT?

File: test
Comment (frzifus) [inline test:1]: one last thing, do we need this file? :baby_chick:

Comment (rubenvp8510) [reply test:1]: not really :D

**Review by frzifus — APPROVED** [review]

File: .ci/create-release-github.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+OPERATOR_VERSION=$(git describe --tags)
```
Comment (pavolloffay) [inline .ci/create-release-github.sh:3]: There is really no need for this script and it can be directly done in the GH action step.

**Pull request merged**